README.md

```diff
@@ -499,20 +545,6 @@ pairwise constructs as found in e.g. `let` and `cond`.
   Avoid the use of overly long namespaces (i.e., more than 5 segments).
 <sup>[[link](#namespaces-with-5-segments-max)]</sup>
 
-* <a name="10-loc-per-fn-limit"></a>
```

Comment:
I think it's pretty reasonable to keep this, as good practice. The same applies for forward references.